src/styles/styles.css

```diff
@@ -1,4 +1,8 @@
 
 body {
     margin: 0;
+}
+
+.bg-custom {
+    background-color: #F1F1F1 !important;
```

Comment:
important overwrites all the css properties - So use it when there is no other alternative.